README.md

```diff
@@ -1796,12 +1799,12 @@ console.log(await client.marginRepay({ asset: 'BTC', amount:'0.0001' }));
 Query Isolated Margin Account Info
 
 ```js
-console.log(await client.marginIsolatedAccount({ symbol: 'BTCUSDT'}));
+console.log(await client.marginIsolatedAccount({ symbols: 'BTCUSDT'}));
```

Comment:
Hum might need to have a breaking change on that, so rather hold on for now

Reply:
@balthazar 
it was my error from one of the previous PR. Here is docs https://binance-docs.github.io/apidocs/spot/en/#query-isolated-margin-account-info-user_data

Do you want a separate PR for it?